### PR TITLE
create details modal of the positions table

### PR DESCRIPTION
**Pull request by ANGIECATHERINEP**
create details modal of the positions table

### CODE REVIEW

**Comment by ANGIECATHERINEP** [comment]
The error that is appearing in the Details Modal component (about the lack of selectedItem property) was adjusted by Lincon in set 794

**Review by LinconOssa — APPROVED** [review]
It happens but remember that in order for us to see the details here we have to wait for card 794 to be accepted to see that everything is fine.

File: src/pages/privileges/outlets/positions/components/DetailsModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <>
+      <Icon
+        icon={<MdOutlineAssignmentTurnedIn />}
+        size="16px"
+        appearance="dark"
+        onClick={handleToggleModal}
+        cursorHover
+      />
+      {showModal && data && (
+        <InteractiveModal
```
Comment (carin91) [inline src/pages/privileges/outlets/positions/components/DetailsModal/index.tsx:30]: The Interactive Component is broken due to interventions for SearchUserCard, a new card was created to solve this issue. After this, you could complete this card.

File: src/pages/privileges/outlets/positions/config/dataPositions.tsx [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
 import { Icon } from "@inube/design-system";
 
 import { mockRoles } from "@mocks/privileges/roles/Roles.mock";
 
 import { activateRoleModal } from "../../roles/config/activateRole.config";
 import { ActivateFormOptions } from "../../forms/ActivateFormOptions";
+import { DetailsModal } from "../components/DetailsModal";
+
+const dataDetailsRol = (k_Rol: string) => {
+  const data = [mockRoles.find((role) => role.k_Rol === k_Rol)!].map(
```
Comment (carin91) [inline src/pages/privileges/outlets/positions/config/dataPositions.tsx:12]: It is much better to solve the roles vs positions issue before everything. Solve first this card, to complete the Details Modal card.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/pages/privileges/outlets/positions/config/dataPositions.tsx [hunk 3]
```diff
@@ -30,6 +26,19 @@ export const PositionsBreakPointsConfig = [
   { breakpoint: "(max-width: 430px)", totalColumns: 1 },
 ];
 
+const dataDetailsPosition = (k_Grupo: string) => {
+  const data = [
+    MockPositions.find((position) => position.k_Grupo === k_Grupo)!,
```
Comment (carin91) [inline src/pages/privileges/outlets/positions/config/dataPositions.tsx:31]: You must look into the positions database, not the mockPositions object. Refactor to find in the "linix-positions" mock.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/pages/privileges/outlets/positions/config/dataPositions.tsx (hunk 3 above)
Comment (ANGIECATHERINEP) [reply src/pages/privileges/outlets/positions/config/dataPositions.tsx]: resolved

**Review by carin91 — APPROVED** [review]

**Pull request merged**